build.gradle

```diff
@@ -18,13 +18,16 @@ ext {
     cglibVersion = "3.2.0"
     objenesisVersion = "2.2"
     jaxbVersion = "2.2.11"
+    betamaxVersion = "2.0.0-alpha-1"
```

Comment:
What is `betamaxVersion` used for? Seems to me that only `betamaxSnapshotVersion` is used.